lib/crypto.js

```diff
@@ -21,7 +21,7 @@ const timingSafeEqual = binding.timingSafeEqual;
 const Buffer = require('buffer').Buffer;
 const stream = require('stream');
 const util = require('util');
-const LazyTransform = require('internal/streams/lazy_transform');
+const LazyTransform = require('internal/crypto/lazy_transform');
```

Comment:
The lazy_transform internal class is only used by the crypto module. I discussed this with @zvictor at the code and learn this last weekend and we went through and made sure. I'm happy with moving it in order to make the relationship between this class and the crypto module clear.